Sources/PT.PM.Matching/Patterns/PatternStringRegexLiteral.cs

```diff
@@ -36,13 +36,15 @@ public override MatchContext Match(StringLiteral stringLiteral, MatchContext con
             if (stringLiteral.InitialTextSpans.Any())
             {
                 List<TextSpan> result = new List<TextSpan>();
-                stringLiteral.InitialTextSpans.Sort();
+                var initialTextSpans = stringLiteral.InitialTextSpans.OrderBy(el => el.Start).ToList();
```

Comment:
`TextSpan` implement `IComparable` interface. Why do not to use just `el => el`?